src/commands/env/create/compute.ts

```diff
@@ -46,7 +46,14 @@ export default class EnvCreateCompute extends Command {
     const alias = flags.alias;
 
     // if `--connected-org` is null here, fetchOrg will pull the default org from the surrounding environment
-    const org = await fetchOrg(flags['connected-org']);
+    let org: Org | null;
```

Comment:
I don't think the union null is needed here.

```suggestion
let org: Org;
```